packages/graphback/src/resolvers/resolverTemplate.ts

```diff
@@ -1,7 +1,11 @@
 import { TargetResolverContext } from './knex/targetResolverContext';
 
+const imports = `import { Context } from '../src/context'`
```

Comment:
Doing the import here

Reply:
Rename it to something like GeneratedContext or something similar.